iidm/iidm-xml-converter/src/test/java/com/powsybl/iidm/xml/TerminalMockXmlSerializer.java

```diff
@@ -66,7 +66,8 @@ public void write(TerminalMockExt extension, XmlWriterContext context) throws XM
         NetworkXmlWriterContext networkContext = (NetworkXmlWriterContext) context;
         String extensionVersion = networkContext.getOptions().getExtensionVersion(getExtensionName())
                 .orElseGet(() -> getVersion(networkContext.getVersion()));
-        TerminalRefXml.writeTerminalRef(extension.getTerminal(), networkContext, getNamespaceUri(extensionVersion), "terminal", context.getExtensionsWriter());
+        TerminalRefXml.writeTerminalRef(extension.getTerminal(), networkContext, getNamespaceUri(extensionVersion),
+                "terminal", "", context.getExtensionsWriter());
```

Comment:
I think it would be clearer to use a method without an `id` parameter (`TerminalRefXml.writeTerminalRef(extension.getTerminal(), networkContext, getNamespaceUri(extensionVersion),
                "terminal", context.getExtensionsWriter());`) which calls `TerminalRefXml.writeTerminalRef(extension.getTerminal(), networkContext, getNamespaceUri(extensionVersion),
                "terminal", null, context.getExtensionsWriter());` and then putting the condition in the method:
```java
            if (id != null && !id.equals(c.getId())) {
                writeId[0] = true;
            }
```